src/test/java/com/eleks/academy/whoami/core/state/AbstractGameStateTest.java

```diff
@@ -23,9 +22,9 @@ void findPlayerTest() {
 
 		players.put(player, new PersistentPlayer(player));
 
-		var findPlayer = state.findPlayer(player);
-		var newPlayer = new PersistentPlayer(player);
-		Optional<SynchronousPlayer> expectedPlayer = Optional.of(newPlayer);
+		var findPlayerOp = state.findPlayer(player);
```

Comment:
@Flozik look: `assertEquals(new PersistentPlayer(player), state.findPlayer(player).get());` and I even do not create variables, variables are only needed when you going to use them somewhere if you can do without them do it